.github/scripts/install-torch-tensorrt.sh

```diff
@@ -1,13 +1,18 @@
-#!/usr/bin/env bash
 set -eou pipefail
-# Source conda so it's available to the script environment
-source ${BUILD_ENV_FILE}
-export EXTRA_INDEX_URL="https://download.pytorch.org/whl/nightly/${CU_VERSION}"
+
+TORCH_TORCHVISION=$(grep "^torch" ${PWD}/py/requirements.txt)
+INDEX_URL=https://download.pytorch.org/whl/${CHANNEL}/${CU_VERSION}
+PLATFORM=$(python -c "import sys; print(sys.platform)")
 
 # Install all the dependencies required for Torch-TensorRT
-${CONDA_RUN} pip install --pre -r ${PWD}/tests/py/requirements.txt --use-deprecated=legacy-resolver --extra-index-url=${EXTRA_INDEX_URL}
+pip install --pre ${TORCH_TORCHVISION} --index-url ${INDEX_URL}
+pip install --pre -r ${PWD}/tests/py/requirements.txt --use-deprecated legacy-resolver
 
 # Install Torch-TensorRT
-${CONDA_RUN} pip install /opt/torch-tensorrt-builds/torch_tensorrt*.whl
```

Comment:
Could the `${CONDA_RUN}` be added back, or is there a specific reason it was removed? It seemed helpful for ensuring correct Python versioning, and would be good to include if it does not break anything